deploy/examples/data-index.yaml

```diff
@@ -19,7 +19,7 @@ spec:
   # details about the kafka connection
   kafka:
     # the service name and port for the kafka cluster. Example: my-kafka-cluster:9092
-    serviceURI: kogito-kafka-kafka-bootstrap:9092
+    externalURI: kogito-kafka-kafka-bootstrap:9092
```

Comment:
We might stick with  `instance: kogito-kafka` instead.

Reply:
done